post-result.py.in

```diff
@@ -4,32 +4,215 @@ import os
 import subprocess
 import sys
 
```

Comment:
There is a script build-config which replace the environment variables in file .in with values and generate file .py